packages/manager/modules/pci/src/components/users/download-rclone/download-rclone.controller.js

```diff
@@ -10,10 +14,14 @@ export default class PciUsersDownloadRcloneController {
     this.$compile = $compile;
     this.$translate = $translate;
     this.ovhManagerRegionService = ovhManagerRegionService;
+    this.downloadFileType = DOWNLOAD_FILETYPE;
   }
 
   $onInit() {
     this.isLoading = false;
+    this.fileType = DOWNLOAD_FILETYPE.SWIFT;
+    this.swiftFileType = DOWNLOAD_FILETYPE.SWIFT;
+    this.s3FileType = DOWNLOAD_FILETYPE.S3;
```

Comment:
**nitpick:** Since the entire constant is associated to controller instance on line 17, this can be removed.

Reply:
Okay Anoop